cleaned_data/modsbyPID/rftaart:12.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mods xmlns="http://www.loc.gov/mods/v3"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xs="http://www.w3.org/2001/XMLSchema"
+      xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+   <identifier type="local">RFTA_Daniel_001</identifier>
+   <identifier type="pid">rftaart:12</identifier>
+   <titleInfo>
+      <title>Smokey the Bear sends a message to the victims</title>
+   </titleInfo>
+   <originInfo>
+      <dateCreated>2016-12-01</dateCreated>
+      <dateCreated encoding="edtf" keyDate="yes">2016-12-01</dateCreated>
+   </originInfo>
+   <abstract>Hats off to all the tireless firefighters and the undaunted spirit of Tennesseans</abstract>
+   <name>
+      <namePart>Daniel, Charlie</namePart>
```

Comment:
@logantillman - Charlie Daniel has existing records in our collection with his name in a particular form. Can you change this value to "Daniel, Charles R. (Charlie), Jr., 1930-". This will ensure that all the records will appear under the same name in a search facet. It looks like we've never established a formal authority for Charlie Daniel, but I will request that we do and we can include this in the future.